README.markdown

```diff
@@ -38,9 +44,9 @@ bootstrapped:
 
     git clone git://github.com/rspec/rspec-dev.git
     cd rspec-dev
-    bundle install --binstubs
-    bin/rake setup
-    bin/rake # runs tests in every repository
+    bundle install --path=vendor/bundle
```

Comment:
I just tried the `rubygems-bundler` gem (suggested in [this two-year-old stackoverflow answer](http://stackoverflow.com/questions/11922386/railss-bundle-binstubs-on-windows)).  It doesn't work.